src/api/app/models/bs_request.rb

```diff
@@ -1,6 +1,7 @@
 require 'xmlhash'
 require 'event'
 require 'opensuse/backend'
+require File.join(Rails.root, "lib/workers/accept_requests")
```

Comment:
Erm. why not just `require 'workers/accept_requests'` like the other files? `lib` is in the autoload path